notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -55,10 +55,10 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    compression_mode
</pre><p>can we convert several converted precision in once or only one always should can be selected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Yes, it would just be an extra work considering that only one of the model will be used in the end. 

Do you suggest to replace dropdown menu with multiple checkboxes?

Reply:
I'm not sure, does it makes sense? From comparison point of view it will be convenient to compress models at once, but if later we need to select anyway which model we want to use, I think it maybe more stright-forward to use your proposed style with only single option without extra complicated logic

Reply:
<p>Sometime it makes sense and doesn't bring any complications for uses that don't need it. I'll add it. :+1:</p>